### PR TITLE
KOGITO-2810: In online editor, the filename in header toolbar is not accessible

### PR DESCRIPTION
**Pull request by ljmotta**
## Task
https://issues.redhat.com/browse/KOGITO-2810

## On this PR
- Remove the file extension from the file name.
- Make it possible to edit the file name without the necessity of a mouse.

## Demo
ezgif com-video-to-gif (5)

### CODE REVIEW

**Comment by srambach** [comment]
This looks much better! Thanks for addressing it. 🎉

**Review by tiagobento — APPROVED** [review]

File: packages/online-editor/src/editor/EditorToolbar.tsx [hunk 1]
```diff
@@ -148,34 +142,28 @@ export function EditorToolbar(props: Props) {
...
+        />
+      </div>
+      {props.isEdited && (
+        <span
+          aria-label={"File was edited"}
+          className={"kogito--editor__toolbar-edited"}
+          data-testid="is-dirty-indicator"
+        >
+          {` - ${i18n.terms.edited}`}
+        </span>
```
Comment (jomarko) [inline packages/online-editor/src/editor/EditorToolbar.tsx:166]: Could we please add unit test that confirms dirty indicator is rendered?

**Review by jomarko — COMMENTED** [review]
Manually checked on chrome and firefox. Can confirm works without an issue. Thank you for this contribution. 

Single comment to coverage.

File: packages/online-editor/src/editor/EditorToolbar.tsx (hunk 1 above)
Comment (ljmotta) [reply packages/online-editor/src/editor/EditorToolbar.tsx:166]: Hi @jomarko . There already exists unit tests regarding the dirty indicator on [EditorToolbar.test.ts](https://github.com/kiegroup/kogito-tooling/blob/master/packages/online-editor/src/__tests__/editor/EditorToolbar.test.tsx).

**Comment by ederign** [comment]
@jomarko as you kind approve this and this is a blocker issue I'll merge this to include 0.6.1. If the tests that @ljmotta are not enough, please let us know and we include ASAP.

**Review by ederign — APPROVED** [review]

**Pull request merged**